avm/res/document-db/database-account/tests/e2e/autoFailoverOff/main.test.bicep

```
@@ -0,0 +1,77 @@
+targetScope = 'subscription'
+
+metadata name = 'autoFailoverOff'
+metadata description = 'This instance deploys the module disabling automatic failover.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-documentdb.databaseaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dddaauo'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
+var enforcedLocation = 'eastus'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, enforcedLocation)}-nestedDependencies'
+  params: {
+    location: enforcedLocation
+    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
+    pairedRegionScriptName: 'dep-${namePrefix}-ds-${serviceShort}'
+  }
+}
+
+// ============== //
+// General resources
+// ============== //
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2022-09-01' = {
+  name: resourceGroupName
+  location: enforcedLocation
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module disableAutomaticFailover '../../../main.bicep' = {
```

```suggestion
module testDeployment '../../../main.bicep' = {
```
That's why the script didn't work. As per the specs (a bit hidden maybe) it must be the above name to identify the relevant test 'block'